src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -3,6 +3,8 @@ import { orderBy } from 'natural-orderby';
 import * as path from 'path';
 import * as request from 'request';
 import * as rokuDeploy from 'roku-deploy';
+// eslint-disable-next-line
+const Telnet = require('telnet-client');
```

Comment:
does this need to be imported anymore? It doesn't appear to be used anywhere in this file.

Reply:
Nope. Will remove.